recipe/meta.yaml

```diff
@@ -28,17 +28,19 @@ requirements:
     - wheel
   run:
     - python
-
     - affine
     - attrs
+    - boto3 >=1.3.1
```

Comment:
Just wondering where you got the boto3 and pyparsing pinnings from? The install_requires in setup.py define the runtime requirements, and they're defined [here](https://github.com/rasterio/rasterio/blob/560d4dd6b6c7fb402779ac815f83d618f357d013/setup.py#L264C1-L264C10) for this package

Reply:
My mistake, boto3 is a required dep for testing, not run dependencies. I'll make an amendment to that. Pyparsing was a requirements that was also suggested for testing [here](https://github.com/rasterio/rasterio/blob/24d0845e576158217f6541c3c81b163d873a994d/requirements.txt#L11)